pkg/agent/route/route_windows.go

```diff
@@ -48,6 +49,15 @@ func NewClient(hostGateway string, serviceCIDR *net.IPNet, encapMode config.Traf
 // Service LoadBalancing is provided by OpenFlow.
 func (c *Client) Initialize(nodeConfig *config.NodeConfig) error {
 	c.nodeConfig = nodeConfig
+	// Enable IP-Forwarding on the interface of OVS bridge, and the host networking stack can be used to forward the
+	// SNAT packet from local Pods. The SNAT packet is output to the OVS bridge interface with the Node's IP as the
+	// the src IP, the external address as the dst IP, and the gw0's MAC as the dst MAC. After the bridge interface gets
+	// the packet, it forwards the packet on the host networking, and the packet's dst MAC could be reset with a correct
```

Comment:
Should we say "The packet will be forwarded to the host network stack from the bridge interface, and its dst MAC could be resolved to the right one".